openverse_catalog/dags/common/storage/media.py

```diff
@@ -142,6 +144,9 @@ def clean_media_metadata(self, **media_data) -> Optional[dict]:
 
         media_data.pop("license_info", None)
         media_data["provider"] = self.provider
+        if not media_data["category"]:
+            if provider_category := prov.DEFAULT_IMAGE_CATEGORY.get(self.provider):
+                media_data["category"] = provider_category
```

Comment:
Could this just be?
```suggestion
        media_data["category"] = media_date["category"] or prov.DEFAULT_IMAGE_CATEGORY.get(self.provider)
```

This will still set `media_data["category"]` to `None` if there is neither a category for that record or a default provider category.